### ISSUE TITLE
doc on pythonhosted.org is not searchable

### ISSUE THREAD

**Issue by vlcinsky** [mozman/ezdxf#16]
There are two documentation sites:

- http://ezdxf.readthedocs.io/en/latest/ - the "readthedocs"
- http://pythonhosted.org/ezdxf/ - the "pythonhosted"

Trying to search (using the builtin Sphinx based search running within browser):
- the "readthedocs" works as expected
- while the "pythonhosted" does not bring any results.

Proposed solution:
- change README to show only the readthedocs site as it works (now it first shows the "pythonhosted"
  one)
- if possible, fix the "pythonhosted" site (no idea, what can cause this), otherwise stop declaring
  this doc url.

**Comment by mozman** [comment]
Maybe a browser problem, I can search at pythonhosted.org

**Comment by vlcinsky** [comment]
I have tested it on:

- Firefox v53: Ubuntu 16.04: http://whatsmybrowser.org/b/4ZHX70J
- Chromium: Ubuntu 16.04: http://whatsmybrowser.org/b/RUSX5D6
- Firefox on Android
- Windows 10: Chrome 58

In all cases it shows described behaviour.

Other observation: searching term "insert" fails in pythonhosted, but without ending "t" thus "inser" it finds something. Similarly "sectio" works, but "section" does not (on pythonhosted).

I used browser inspector (F12) and did not find anything, what would look suspicious (but did not dive too deep).
